.eslintignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
The negative sign means you need a new line at the end of the file. 😄

Reply:
Can you please let us know more about this? We don't see any negative sign on the VSCode.

Reply:
It only shows when you click on the `Files changed` section of the PR and look at the .eslintignore file. You will see a circle ⭕️ with a minus sign inside of it meaning that you did add a new line at the end of the file.